functions/nvm.fish

```diff
@@ -4,7 +4,7 @@ function nvm
     return
   end
   set -q NVM_DIR; or set -gx NVM_DIR ~/.nvm
-  set -g nvm_prefix $NVM_DIR
+  set -q nvm_prefix; or set -g nvm_prefix $NVM_DIR
```

Comment:
could you just add the `-gx` so it gets exported to the child processes?

`set -q nvm_prefix; or set -gx nvm_prefix $NVM_DIR`

Reply:
Sorry, my bad.

Added -gx parameters.

Reply:
Sorry, my bad.

Added -gx paramaters